features/polyExplorer/src/locales/de/stories/clusterDigitalGiants.json

```diff
@@ -4,6 +4,11 @@
     "intro.p.1": "Wenn Sie heutzutage ein digitales Leben führen, ist es fast unmöglich, nicht mit einigen der großen Technologieunternehmen und ihren Diensten in Kontakt zu kommen. Was diese Unternehmen gemeinsam haben, ist ihr enormer Appetit auf Daten und die Absicht, diese Daten - wahrscheinlich Ihre Daten - für Werbung oder anstehende Big-Data-Projekte zu nutzen und zu Geld zu machen.",
     "intro.p.2": "Das polyPedia-Team hat sich alles angesehen, was die unten aufgeführten Unternehmen über ihre Datenschutzbestimmungen sagen.",
     "data.types.p": "Einige Datentypen müssen für das, was wir von einem Boten erwarten, gemeinsam genutzt werden. Aber nicht alle Boten teilen die gleiche Menge an Datentypen. Die zusätzliche gemeinsame Nutzung zeigt, dass sie damit Geld verdienen.",
+    "datatypes.legend": "data type",
```

Comment:
Probably not important, but you're using `data.types` above.

Reply:
I didn't see that - different people adding content in the same place. It doesn't really make any difference, but I'll change it for consistency.